### DISCUSSION TITLE
Request to join team.

### DISCUSSION THREAD

**Discussion by Saikrishna1504** [ROM-builders/temporary#21863] [General]
Successful Builds:-
Voltage os (official) : https://cirrus-ci.com/build/5361911974330368
Alpha Droid (unofficial) : https://cirrus-ci.com/build/5365518203355136

My Device trees : https://github.com/Saikrishna1504/android_device_redmi_begonia
And our organisation : https://github.com/Trinity-ROMS

Telegram I'd : https://t.me/Saikrishna1504

**Comment by Apon77** [comment]
Please follow #learn instruction from telegram group.

**Reply by Saikrishna1504** [reply]
https://github.com/Saikrishna1504/android_device_redmi_begonia
Successfull builds:

Voltage(official) : https://cirrus-ci.com/build/5361911974330368
Alpha (unofficial) : https://cirrus-ci.com/build/5365518203355136

Tg: https://t.me/Saikrishna1504
Can I join now bro ?
